espressif/idf-eclipse-plugin
# fix: Open .md files in the eclipse internal text editor

## Description

Open .md files in the eclipse text editor instead of trying to open in the external system editor.

Fixes # ([IEP-1075](https://jira.espressif.com:8443/browse/IEP-1075))

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How has this been tested?

Test case 1:
- Download espressif-ide
- Create a project
- Open a README.md file and see if it's able to open with in the eclipse text editor

Test case 2:
- Update eclipse cdt with the PR update site
- Create a project
- Open a README.md file and see if it's able to open with in the eclipse text editor

**Test Configuration**:
* ESP-IDF Version:
* OS (Windows,Linux and macOS):

## Dependent components impacted by this PR:

- Update site
- Espresif-IDE
- Opening markdown files

## Checklist
